### ISSUE TITLE
Julian Day issue

### ISSUE THREAD

**Issue by Alexander-Barth** [Unidata/cftime#52]
I am getting strange results with the function `cftime.DateFromJulianDay` using the Gregorian Calendar:

```python
cftime.DateFromJulianDay(1684958.5,calendar='gregorian')
cftime.DatetimeProlepticGregorian(-100, 3, 2, 23, 59, 59, 999967, -1, 1)
```
Thus effectively, the 3rd March, -100 BC.
A very small change to the Julian Day, give the 2nd March, -100 BC.

```python
cftime.DateFromJulianDay(1684958.50001,calendar='gregorian')
cftime.DatetimeProlepticGregorian(-100, 3, 2, 0, 0, 0, 863974, -1, 1)
```

The issue also affect the standard calendar:

```python
cftime.DateFromJulianDay(1684958.5,calendar='standard')
# returns cftime.DatetimeGregorian(-100, 3, 2, 23, 59, 59, 999967, -1, 1)
cftime.DateFromJulianDay(1684958.50001,calendar='standard')
# return cftime.DatetimeGregorian(-100, 3, 2, 0, 0, 0, 863974, -1, 1)
```

According to this converting web-site (http://aa.usno.navy.mil/jdconverter?ID=AA&jd=1684958.5), 2nd March, -100 BC should be correct.

I thing that cftimes uses the Jean Meeus algorithm, which should be correct for negative years (as long as the JD is positive).

**Comment by Alexander-Barth** [comment]
Maybe this is related?

```python
JD = 1684959.50001
dt = cftime.DateFromJulianDay(JD,calendar='proleptic_gregorian')
cftime.JulianDayFromDate(dt) - JD
# returns -2.0
```

I use cftime version 1.0.0 and python 2.7.12.

**Comment by jswhit** [comment]
I think this is fixed by pull request #65

**Comment by jswhit** [comment]
fixed now in master, closing
